spec/integration/balancing_spec.rb

```diff
@@ -173,9 +173,8 @@
     expect(page).to have_selector('form#new_order_article')
     within('#new_order_article') do
       find('#order_article_article_id').select(new_article.name)
-      sleep 0.1
+      sleep 0.25
```

Comment:
It's just this one I focused on, and made everything `0.25` just as a bit extra. Let's add more when we see it's necessary.